src/core/Akka.Docs.Tutorials/Akka.Docs.Tutorials.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+    <Import Project="..\..\common.props" />
+  <PropertyGroup>
+      <AssemblyTitle>Akka.Docs.Tutorials</AssemblyTitle>
+      <TargetFrameworks>$(NetFrameworkTestVersion);$(NetCoreTestVersion)</TargetFrameworks>
+      <RootNamespace>Tutorials</RootNamespace>
```

Comment:
Need to add the following here:

`<IsPackable>false</IsPackable>`

That will stop us from accidentally publishing an Akka.Docs.Tutorials NuGet package

Reply:
Ok, done.